html/templates/header.html

```diff
@@ -23,26 +23,25 @@
+			<p><div align='center'><font size='3'><a href="https://beestation13.com/">Site</a> | <a href="https://beestation13.com/forum">Forum</a> | <a href="https://wiki.beestation13.com/">Wiki</a> | <a href="https://github.com/BeeStation/BeeStation-Hornet">Source</a></font></div></p>
+			<font size='2'><b>Join our Discord </b><a href="https://discord.gg/Tfxb5TX">Here!</a></font>
 			</td>
 	</tr>
 </table>
 
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<font size='2'><b>Current Project Maintainers:</b> <a href='https://github.com/tgstation?tab=members'>-Click Here-</a><br></font>
-			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/tgstation/-tg-station/graphs/contributors'>-Click Here-</a><br></font>
-			<font size='2'><b>Coders:</b> TLE, NEO, Errorage, muskets, veryinky, Skie, Noise, Numbers, Agouri, Noka, Urist McDorf, Uhangi, Darem, Mport, rastaf0, Doohl, Superxpdude, Rockdtben, ConstantA, Petethegoat, Kor, Polymorph, Carn, Nodrak, Donkie, Sieve, Giacom, Ikarrus, trubble_bass, Aranclanos, Cael_Aislinn, Cheridan, Intigracy, Malkevin, SuperSayu, DumpDavidson, Tastyfish, Yvar, Elo001, Fleure, ManeaterMildred, Miauw, MrPerson<br></font> 
-			<font size='2'><b>Spriters:</b> Agouri, Cheridan, Cruazy Guest, Deeaych, Deuryn, Matty406, Microwave, ShiftyEyesShady, Skie, Uhangi, Veyveyr, Petethegoat, Kor, Ricotez, Ausops, TankNut, Pewtershmitz, Firecage, Nienhaus2<br></font>
-			<font size='2'><b>Sounds:</b> Skie, Lasty/Vinyl<br></font>
-			<font size='2'><b>Main Testers:</b> Tenebrosity, Anyone who has submitted a bug to the issue tracker<br></font>
-			<font size='2'><b>Thanks to:</b> Baystation 12, /vg/station, NTstation, CDK Station devs, FacepunchStation, GoonStation devs, the original SpaceStation developers and Invisty for the title image.<br> Also a thanks to anybody who has contributed who is not listed here :( Ask to be added here on irc.</font>
-			<font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/tgstation/-tg-station/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font>
-			<font size='2'>Please ensure that the bug has not already been reported and <u><a href="https://github.com/tgstation/-tg-station/issues/724"><b>use the template provided here</b></a></u>.</font>
+			<font size='2'><b>Current Project Maintainers:</b> <a href='https://github.com/orgs/BeeStation/people'>-Click Here-</a><br></font>
+			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/BeeStation/BeeStation-Hornet/graphs/contributors'>-Click Here-</a><br></font>
+			<font size='2'><b>Coders:</b> CthuluOnIce, Qwertyquerty, ike709, Crossedfall, AzlanonPC, Mark Suckerberg, Cenrus, St0rmC4st3r, ShadowH4nD, <br></font> 
+			<font size='2'><b>Spriters:</b> Thatguythere03, AzlanonPC, OnePlusOne<br></font>
+			<font size='2'><b>Mappers:</b> MiniusAreas, Leader425, AzlanonPC, Debonaircorsiar, Mark Suckerberg<br></font>
```

Comment:
```suggestion
			<font size='2'><b>Mappers:</b> Prophed<br></font>
```

This is usually reserved for people who have designed and implemented a new map or made large changes to existing maps. Not really for people who make minor map edits. Honestly after removing basically everyone, it might be best to just leave the original sounds credit in (considering we still use those sounds that seems fair). 

Deb - Doesn't have any merged PRs relating to mapping. 
miniusAreas - Doesn't have any PRs at all (merged or not)
Leader425 - I don't see any PRs from them, maybe they have a different username on github?
You - I could be mistaken, but I didn't see any major map edits. I saw feature additions and some minor fixes.

No credit to Prophed? Even though it's not in rotation they did make Kilo.

Reply:
Oh, alright, wasn't aware. Like I said, I have legitimately 0 idea on what to put on this, may just revert it and leave it as what it was before at this rate